.claude/scripts/ai-review/prioritizer.py

```
@@ -0,0 +1,299 @@
+
+    def _check_zero_tolerance(self, description: str) -> str:
+        """
+        Zero-Tolerance 규칙 체크
+
+        Args:
+            description: 이슈 설명
+
+        Returns:
+            위반 시 이유, 아니면 빈 문자열
+        """
+        for rule_name, rule_data in self.ZERO_TOLERANCE_PATTERNS.items():
```

Ruff (0.14.3)

Loop control variable `rule_name` not used within loop body

Rename unused `rule_name` to `_rule_name`

(B007)